index.js

```diff
@@ -46,6 +46,9 @@ module.exports = {
                 ],
                 "@typescript-eslint/no-floating-promises": "error",
                 "@typescript-eslint/comma-dangle": ["error", "always-multiline"],
+                "@typescript-eslint/await-thenable": "error",
+                "@typescript-eslint/no-misused-promises": "error",
```

Comment:
this one is a bit problematic, looks like it will make async event handlers impossible (without eslint ignore comments) - which agreed is a problematic piece of code in the first place, but our code base is full of such snippets. we usually call `catch()` on such promise handlers or ensure they never reject in other ways (e.g. try/catch inside the handler, and those things are still producing errors from this rule

Reply:
FYI, this config is used in the app:
```
"@typescript-eslint/no-misused-promises": [
        "error",
        {
            "checksVoidReturn": {
                "arguments": false,
                "attributes": false
            }
        }
    ],
```
I mean, sure, we can just omit this if it causes problems

Reply:
Nice, with that options I get just a single error, so let's do it like that?